contracts/mocks/MockController.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma experimental ABIEncoderV2;
+
+import {Actions} from "../libs/Actions.sol";
+
+contract MockController {
+    Actions.DepositArgs private depositArgs;
+
+    function testParseDespositAction(Actions.ActionArgs memory _args) external {
+        depositArgs = Actions._parseDepositArgs(_args);
```

Comment:
Can't we just use `return Actions._parseDepositArgs` here and get rid of the `getDepositArgs`? 
This will make the test a little bit easier

Reply:
can't get return data while testing in truffle unless I emit an event. So either I should emit event or have this getter function. To test the state changes I decided to use the getter function. Event may mean that it gets emitted but doesn't change state